src/setupTest.js

```diff
@@ -18,3 +32,60 @@ global.IntersectionObserver = jest.fn(function mockIntersectionObserver() {
 window.getComputedStyle = jest.fn(() => ({
   getPropertyValue: jest.fn(),
 }));
+
+let globalStore;
+
+// eslint-disable-next-line import/prefer-default-export,no-unused-vars
+export async function initializeTestStore(options = {}, overrideStore = true) {
```

Comment:
This is perhaps not very well named - 'store' refers to redux.  The other things this function is doing is unrelated to redux.  

Also, we don't need to use `configureStore` from redux directly - there's a function in store.js that does this so that we don't have to reproduce this code.

What's the purpose of the overrideStore check?  When would you not replace the store?  That seems like asking for problems... every test should clear all state from prior tests; it should never be the case where we persist data between tests.